api/handlers.go

```diff
@@ -80,13 +81,17 @@ func (api *API) loginPOST(w http.ResponseWriter, req *http.Request, _ httprouter
 		api.WriteError(w, err, http.StatusUnauthorized)
 		return
 	}
-	exp, err := jwt.TokenExpiration(token)
-	if err != nil {
-		api.staticLogger.Traceln("Error checking token expiration:", err)
-		api.WriteError(w, err, http.StatusUnauthorized)
+	// We fetch the expiration time of the token, so we can set the expiration
+	// time of the cookie to match it.
+	exp := token.Expiration()
+	if time.Now().UTC().After(exp) {
```

Comment:
I think you're off here with the timezones because 4 lines below you do `exp.UTC()`.
So it should probably be 
```
if exp.UTC().before(time.Now().UTC()) {
    // handle expired token
}
```
is that  possible?

Reply:
Yes, well spotted! Thanks!